src/sentry/deletions/defaults/group.py

```diff
@@ -107,10 +107,12 @@ def get_child_relations(self, instance):
         return relations
 
     def delete_instance(self, instance):
-        from sentry.similarity import features
+        from sentry import similarity
 
-        if not self.skip_models or features not in self.skip_models:
-            features.delete(instance)
+        assert not self.skip_models or similarity.features not in self.skip_models
```

Comment:
delete this assertion after CI has passed, just wanted to make sure this works because passing in the features instance as skip_models is some sus shit